.vscode/settings.json

```diff
@@ -6,4 +6,15 @@
 
     // Use 'prettier-eslint' instead of 'prettier'. Other settings will only be fallbacks in case they could not be inferred from eslint rules.
     "prettier.eslintIntegration": true,
+    "search.exclude": {
+        "**/bundle/android/App.js": true,
+        "**/bundle/android/App.js.map": true,
+        "**/bundle/ios/App.js": true,
+        "**/bundle/ios/App.js.map": true,
+        "**/gutenber/packages/**/build": true,
+        "**/gutenber/packages/**/build-modules": true,
+        "**/gutenber/packages/**/build-types": true,
```

Comment:
Is `gutenber` a typo perhaps, missing the final `g`?

Reply:
Yes that was a typo. Thanks for catching it.